bin/asinit

```diff
@@ -8,6 +8,30 @@ const colors = require("../cli/util/colors");
 const version = require("../package.json").version;
 const options = require("../cli/util/options");
 
+const commands = {
+  "npm": {
+    install: "npm install",
```

Comment:
```suggestion
    install: "npm i",
```

Reply:
Same thing - for a bootstrap, I'd suggest to be explicit by default; users can then optimize as needed.